scapy/packet.py

```diff
@@ -1256,7 +1263,7 @@ def ls(obj=None):
                     class_name += " (%d bit%s)" % (cur_fld.size,
                                                    "s" if cur_fld.size > 1
                                                    else "")
-                print "%-10s : %-25s =" % (f.name, class_name),
+                print "%-10s : %-25s =" % (f._name, class_name),
```

Comment:
This will not work, as a field has no attribute `_name` but `name`.

Can you `commit --amend` the fix and then `push -f`?